types/yosensi/YO-PurePro/meta.json

```diff
@@ -0,0 +1,37 @@
+{
+  "name": "YO Pure Pro",
+  "version": "1.0.0",
+  "manufacturer": "Yosensi",
+  "url": "https://www.yosensi.io/products/yo_pure_pro/",
+  "description": "YO Pure Pro is a device for measuring indoor environmental conditions.",
+  "author": "Maciej Sacewicz",
+  "firmwareVersion": "V3.6.3",
+  "availableSensors": [
+    "Internal Temperature",
+    "Relative Humidity",
+    "Atmospheric Pressure",
+    "Illuminance",
+    "TVOC",
+    "CO",
+    "CO2",
+    "Sound Pressure Level",
+    "PM2.5",
+    "PM4",
+    "PM10"
+  ],
+  "outputTopics": [
```

Comment:
Currently a topic is used for each data point. We recommend grouping data points that belong together and are sent in the same LoRa uplink into the same topic; e.g. topic `iaq` containing tvoc, co, co2, or `environment` containing all data keys. We usually distinguish between a `lifecycle` topic containing system information and a measurement topic containing the values of interest.

Reply:
I've been trying to handle varying payloads from our device, but I'm running into issues with passing the tests. For instance, our payloads can sometimes look like this:
Payload 1: 'co2', 'pm10'
Payload 2: 'co', 'pm4', 'sound pressure', ...
Payload 3: 'co'

I haven't been able to come up with a solution that manages this variability effectively, so I ended up handling it in a different way. If you have any tips or ideas on how to better manage these diverse payloads, I'd really appreciate it.

Reply:
ok thanks for the clarification, in this case it makes sense to split the payloads